### DISCUSSION TITLE
Configuration ENOENT question

### DISCUSSION THREAD

**Discussion by bruff-orbitfab** [microsoft/vscode-cmake-tools#4185] [Q&A] [answered]
Question with v1.19.52.  I am seeing 2 ENOENT errors while running configuration.  This is a cross-compilation.  The compiler config succeeds; what are the subsequent failing [proc] calls doing and what paths are being used?

The arm-none-eabi compilers aren't on the system path and are being specified either in CMakeLists.txt or selecting kits.

```
[cmake] -- Configuring done (0.1s)
[cmake] -- Generating done (0.0s)
[cmake] -- Build files have been written to: /Users/...
[proc] The command: arm-none-eabi-gcc -v failed with error: Error: spawn arm-none-eabi-gcc ENOENT
[proc] The command: arm-none-eabi-g++ -v failed with error: Error: spawn arm-none-eabi-g++ ENOENT
```

**Comment by gcampbell-msft** [comment, marked as answer]
@bruff-orbitfab Could you please create an issue if you are still seeing this? Please provide a full repro and we will try to investigate and solve your problem. Thanks!
